Configuration/StandardSequences/python/Services_cff.py

```diff
@@ -13,11 +13,14 @@
 def _addProcessAccelerators(process):
     process.load("Configuration.StandardSequences.Accelerators_cff")
 
+def _addProcessAcceleratorsAlpaka(process):
+    process.load("HeterogeneousCore.AlpakaCore.ProcessAcceleratorAlpaka_cfi")
+
 from Configuration.ProcessModifiers.gpu_cff import gpu
 from Configuration.ProcessModifiers.pixelNtupletFit_cff import pixelNtupletFit
 from Configuration.ProcessModifiers.alpaka_cff import alpaka
 
-modifyConfigurationStandardSequencesServicesAddProcessAccelerators_ = (gpu | pixelNtupletFit | alpaka).makeProcessModifier(_addProcessAccelerators)
+modifyConfigurationStandardSequencesServicesAddProcessAcceleratorsAlpaka_ = alpaka.makeProcessModifier(_addProcessAcceleratorsAlpaka)
```

Comment:
Sorry, I have no idea what this is supposed to do :-(
@makortel ?

Reply:
Maybe this change predates https://github.com/cms-sw/cmssw/pull/43229 that made `ProcessAcceleratorAlpaka_cfi` to be loaded unconditionally in `Accelerators_cff`? Or, I can't think of any way how this specific change would be useful (in fact I'd expect things to break).

Anyway, I think this file should be left untouched by this PR.

Reply:
It does break things :-D
I'll revert it.